apps/teams-test-app/src/components/AuthenticationAPIs.tsx

```diff
@@ -128,6 +128,37 @@ const Authenticate = (): React.ReactElement =>
     },
   });
 
+const externalAuthParams = {
+  url: 'auth_start.html?oauthRedirectMethod=deeplink&authId=1',
```

Comment:
Lets not hardcode these param, especially authId. Also, In android (including Teams), We use a URL which is placeholded at the Hub: like
https://teams-test-tab.azurewebsites.net/auth_start.html?authId={authId}&oauthRedirectMethod={oauthRedirectMethod}

Reply:
For the scenario test I have to hard code the authId because the scenario test cannot control the external browser, so after the external browser is opened, I have to manually raise the onExternalAuthDeeplink event to test the 2nd part of the process. But if we generate a new authId when we open the external browser, we won't be able to get that id when raising the event, so for the purpose of the scenario test, I need to hard code the authId

Reply:
Can you try looking into to mimic that in a reliable manner? For eg. updating a file from auth_end.html and listening/reading that from cypress

Reply:
updated, changed the url to [https://teams-test-tab.azurewebsites.net/auth_start.html?authId={authId}&oauthRedirectMethod={oauthRedirectMethod}](https://teams-test-tab.azurewebsites.net/auth_start.html?authId=%7BauthId%7D&oauthRedirectMethod=%7BoauthRedirectMethod%7D)

Reply:
I would say saving the id somewhere and retrieve it in some manner would require even more uncontrollable sources to work together correctly (network locations, access...), for this test a hardcode id in this case should be an acceptable compromise